docs/wxm-devices/deployment-examples.mdx

```diff
@@ -146,7 +146,7 @@ The trees at the west and south have a significant impact on wind speed/directio
 ### Case 7
 
 <div class="explanation">
-    <img src={useBaseUrl("img/deployments/ZoomBaby.jpg")} />
```

Comment:
You already have referenced `bad_example_7.png` above, so you need to continue with `bad_example_8.jpg` here and add 1 to all other images below.

Reply:
fixed